packages/rrweb-snapshot/test/css.test.ts

```diff
@@ -119,6 +119,83 @@ describe('css parser', () => {
     expect(out3).toEqual('[data-aa\\:other] { color: red; }');
   });
 
+  it.each([
+    ['.foo,.bar {}', ['.foo', '.bar']],
+    ['.bar:has(:disabled) {}', ['.bar:has(:disabled)']],
+    ['.bar:has(input, button) {}', ['.bar:has(input, button)']],
+    [
+      '.bar:has(input:is(:disabled),button:has(:disabled)) {}',
+      ['.bar:has(input:is(:disabled),button:has(:disabled))'],
+    ],
+    [
+      '.bar:has(div, input:is(:disabled), button) {}',
+      ['.bar:has(div,input:is(:disabled), button)'],
+    ],
+    [
+      '.bar:has(div, input:is(:disabled),button:has(:disabled,.baz)) {}',
+      ['.bar:has(div,input:is(:disabled),button:has(:disabled,.baz))'],
+    ],
+    [
```

Comment:
another case, it does seem to fail.
```suggestion
    ],
    [
      '.bar:has(input), .foo:has(input, button), .baz',
      ['.bar:has(input)', '.foo:has(input, button)', '.baz'],
    ],
    [
```

Reply:
i think something is off-by-one, looking ahead at the next segment too soon... not sure though

Reply:
This passes, it's missing curly braces after the selectors

Reply:
d'oh

excellent